test/NuGet.Core.Tests/NuGet.Common.Test/DateTimeUtilityCultureSpecificTests.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using Xunit;
+
+namespace NuGet.Common.Test
+{
+    [Collection(LocalizedTestCollection.TestName)]
+    public class DatetimeUtilityCultureSpecificTests
+    {
+        [Fact]
+        public void ToReadableTimeFormat_NumberFormat_LocaleSensitive_Succeeds()
+        {
+            try
+            {
+                // Prepare
+                CultureUtility.SetCulture(new CultureInfo("es-ES"));
+
+                // Act
+                string actual = DatetimeUtility.ToReadableTimeFormat(TimeSpan.FromSeconds(1.23d));
+
+                // Assert
+                Assert.StartsWith("1,23", actual, StringComparison.InvariantCulture);
+            }
+            finally
+            {
+                LocalizedTestCollection.Reset();
```

Comment:
Instead of calling `.Reset()` at the end of every test, I think xUnit `Collection Fixture` is more appropriate in this context. https://xunit.net/docs/shared-context#collection-fixture. The current culture can be reset to the default culture in the `Dispose` method of the collection fixture only once.

Reply:
Thanks! I will apply on next test. I will remove those tests.